src/Service/ContextTemplateService.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Service;
+
+use Symfony\Component\HttpFoundation\Session;
```

Comment:
Could you include SessionInterface so that you could remove the Session prefix in line 10 & 11?
```suggestion
use Symfony\Component\HttpFoundation\Session\SessionInterface;
```